dead-simple-k8s/hostIP.sh

```diff
@@ -0,0 +1,6 @@
+#!/bin/bash
+
+set -e
+
+sleep 10
```

Comment:
It's a mystery.  It seems like when the container starts, the k8s API isn't guaranteed to already know about it.  If we query the API too soon, this pod isn't there.